app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx

```diff
@@ -95,6 +105,20 @@ const talkDetailsProps = (schedule: ScheduledEvent): TalkDetailsProps => {
       { url: talk?.["speaker-s"][0]?.github, icon: "github" },
       { url: talk?.["speaker-s"][0]?.externalURL, icon: "link" },
     ],
+    isMultipleSpeakers: talk?.["speaker-s"].length > 1,
+    carouselData: talk?.["speaker-s"].map((speaker) => ({
+      image: { uri: speaker?.["speaker-photo"].url },
+      imageStyle: { height: 320 },
+      subtitle: speaker?.name,
+      label: speaker?.company,
+      bodyLabel: `About ${talk?.["speaker-s"][0]["speaker-first-name"]}`,
```

Comment:
This should be `speaker?.["speaker-first-name"]`:
<img width="354" alt="image" src="https://user-images.githubusercontent.com/151139/233470338-923b7c75-e6bc-47f0-b283-aae8656cee0e.png">

I'm not sure that we even need this because the bio appears right below the person's name on these screens. I'd assume that the text was about them.

Reply:
we dont need the `About {firstName}` (a.k.a `bodyLabel`). removed here: https://github.com/infinitered/ChainReactApp2023/pull/155/commits/4110a831c6f90df11ef8e1e9b4b879c93b07ad3f